src/main/game/CommandLine/CommandlineUi.java

```diff
@@ -103,6 +122,23 @@ public void askForLanguage() {
         output.println(english.askForLanguage());
     }
 
+    private String getTranslatedMove(String userMove) {
+        return translations.get(userMove);
+    }
+
+    private boolean languageIsEnglish() {
```

Comment:
I like that you extracted this into a method and you're using it below so 👍 Do you still need these checks however if you're passing the selected language from the start of your program?